sdk/purview/azure-purview-datamap/samples/get_types.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import os
+from azure.identity import ClientSecretCredential
+from azure.purview.datamap import DataMapClient
+
+logging.basicConfig(level=logging.DEBUG)
+LOG = logging.getLogger()
+
+# Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+# AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
```

Comment:
These environment variable names dont match the ones actually used in the sample